### DISCUSSION TITLE
Documentation for V2

### DISCUSSION THREAD

**Discussion by ghost** [tracer-protocol/perpetual-pools-contracts#250] [Ideas]
We have some documentation on Perpetual Pools Contracts V1 on Notion, but we have no public facing documentation outside of Github, we need to scope this for the upcoming V2 release and add this to the board of work to be done.
Need comments on what should be included, please use this thread to discuss @issy-fleming @sporejack @mynameuhh @raymogg @Paul4912 @kumar-ish

**Comment by Paul4912** [comment]
do you mean v1 or v2? title and description says different

**Reply by ghost** [reply]
@Paul4912 For the upcoming V2, sorry for the ambiguity in the original message, I corrected the original entry to be more specific here.
